examples/semantic_segmentation/setup.sh

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Does it have to have two different setup script? I think it's better to have one common compatible setup.sh